libraries/AP_IOMCU/iofirmware/iofirmware.cpp

```diff
@@ -256,13 +264,31 @@ void AP_IOMCU_FW::rcin_update()
         hal.rcout->set_default_rate(reg_setup.pwm_defaultrate);
     }
 
+    // check for active override channel
+    if (mixing.enabled &&
+        mixing.rc_chan_override > 0 &&
+        mixing.rc_chan_override <= IOMCU_MAX_CHANNELS) {
+        override_active = (rc_input.pwm[mixing.rc_chan_override-1] >= 1800);
```

Comment:
Could this remain as 1750 to avoid a behavior change? (A param would work, but sounds excessive)

Reply:
fixed